src/main/java/com/flightstats/hub/app/HubMain.java

```diff
@@ -191,8 +191,8 @@ public Server startServer() throws Exception {
                 services.add(injector.getInstance(S3WriteQueueLifecycle.class));
             }
             services.addAll(createInstanceList(injector,
-                    DynamoChannelConfigDaoLifecycle.class,
-                    DynamoWebhookDaoLifecycle.class));
+                    DynamoChannelConfigExistenceCheckLifecycle.class,
+                    DynamoWebhookExistenceCheckLifecycle.class));
```

Comment:
Yeah, I'll see if I can come up with a better class name.  They're too long.
We keep the concept of webhook and channel config storage separate in other parts of the code, so it seems like these should stay separate here.  They're separate DAOs, etc.

Reply:
When I saw the code in your last PR related to this, I thought it might be a good candidate for a more generic class where you pass the table name in rather than the whole DynamoProperties object. Then we'd need a Provider for each one for guice injection, which I guess isn't particularly elegant.

There's so little code in the two classes that I'm cool with the duplication, though.